tests/TestCase/View/Helper/FlashHelperTest.php

```diff
@@ -4,8 +4,8 @@
 
 use BootstrapUI\View\Helper\FlashHelper;
 use Cake\Core\Exception\Exception;
-use Cake\Network\Request;
-use Cake\Network\Session;
+use Cake\Http\ServerRequest;
+use Cake\Http\Session;
```

Comment:
This change causes tests to fail on 3.5 and we are trying to maintain 3.5 compatibility too. So to make this work on both 3.5 and 3.6 you remove the `use` statement for session class and instead inside `setUp()` use a `class_exists()` check and accordingly use appropriate class to create instance.